README.md

```diff
@@ -1,6 +1,52 @@
-# draive
+# 🏎️ draive 🏁
 
-Framework for building AI oriented applications.
+🏎️ Fast-track your LLM-based apps with an accessible, production-ready library. 🏎️
+
+Are you looking for maximum flexibility and efficiency in your next Python library? Tired of unnecessary complexities and inefficient token usage?
+
+👉 Introducing **draive** - an open-source Python library under the Miquido AI Kickstarter framework, designed to simplify and accelerate the development of LLM-based applications. Get started with draive to streamline your workflow and build powerful, efficient apps with ease.
+
+## 🚀 Quick Install
+
+With pip:
+```bash
+pip install draive
+```
+
+## ❓ What is draive?
+
+**draive** is an open-source Python library for developing apps powered by large language models. It stands out for its simplicity, consistent behavior, and transparency.
+
+### Key Features:
+
+- **🧱 Abstract building blocks**: Easily connect multiple functionalities with LLMs and link various LLMs together.
+- **🧩 Flexible integration**: Supports any LLM, external service, and other AI solutions.
+- **🧒 User-friendly framework**: Designed to build scalable and composable data processing pipelines with ease.
+- **⚙️ Function-oriented design**: Utilizes basic programming concepts, allowing you to represent complex programs as simple functions.
+- **🏗️ Composable and reusable**: Combine functions to create complex programs, while retaining the ability to use them individually.
+- **📊 Diagnostics and metrics**: Offers extensive tools for measuring and debugging complex functionalities.
```

Comment:
please add point about being fully typed and asynchronous

Reply:
done!